### PR TITLE
Fixes #18923

### LINKED ISSUE #18923: You can kick glass cases with T.K from far away
Really makes you think huh?

### PR DESCRIPTION
**Pull request by bobdobbington**
Fixes #18923

### CODE REVIEW

File: code/game/objects/structures/displaycase.dm [hunk 1]
```diff
@@ -191,6 +191,9 @@
 		update_icon()
 		return
 	else
+	    //prevents remote "kicks" with TK
+		if (abs(usr.x-src.x) > 1 || abs(usr.y-src.y) > 1)
```
Comment (optimumtact) [inline code/game/objects/structures/displaycase.dm]: user.adjacent(src) instead 

in general we avoid usr where a proper user variable is available

**Comment by optimumtact** [comment]
I think you need to put exactly

Fixes #xxxxx 

for it to close the issue when merged

**Comment by astralenigma** [comment]
@bobdobbington he meant in the message not the tittle.

**Comment by optimumtact** [comment]
Yes I should have clarified sorry

File: code/game/objects/structures/displaycase.dm [hunk 2]
```diff
@@ -191,6 +191,9 @@
 		update_icon()
 		return
 	else
+	    //prevents remote "kicks" with TK
```
Comment (optimumtact) [inline code/game/objects/structures/displaycase.dm]: What's up with the indentation of this comment?

Comment (MrStonedOne) [inline code/game/objects/structures/displaycase.dm]: its 1 tab + 4 spaces

**Pull request merged**